collector/topological_inventory.py

```diff
@@ -214,25 +219,40 @@ def worker(_: str, source_id: str, dest: str, acct_info: dict) -> None:
     account_id = acct_info['account_id']
     headers = {"x-rh-identity": b64_identity}
 
-    if ALL_TENANTS:
-        resp = utils.retryable('get', TENANTS_URL, headers=headers)
-        tenants_headers = \
-            [tenant_header_info(t["external_tenant"]) for t in resp.json()]
-
-        LOGGER.info('Fetching data for ALL(%s) Tenants', len(tenants_headers))
-
-        for tenant_header in tenants_headers:
-            LOGGER.debug('%s: ---START Account# %s---',
-                         thread.name, tenant_header['acct_no'])
-            headers = tenant_header['headers']
-            topological_inventory_data(_, source_id, dest, headers, thread)
-            utils.set_processed(tenant_header['acct_no'])
-            LOGGER.debug('%s: ---END Account# %s---',
-                         thread.name, tenant_header['acct_no'])
-    else:
-        LOGGER.info('Fetching data for current Tenant')
-        topological_inventory_data(_, source_id, dest, headers, thread)
-        utils.set_processed(account_id)
+    with DATA_COLLECTION_TIME.time():
+        if ALL_TENANTS:
+            prometheus_metrics.METRICS['gets'].inc()
+            resp = utils.retryable('get', TENANTS_URL, headers=headers)
+            prometheus_metrics.METRICS['get_successes'].inc()
+            tenants_headers = \
+                [tenant_header_info(t["external_tenant"]) for t in resp.json()]
+
+            LOGGER.info('Fetching data for ALL(%s) Tenants',
+                        len(tenants_headers))
+
+            for tenant_header in tenants_headers:
+                LOGGER.debug('%s: ---START Account# %s---',
+                             thread.name, tenant_header['acct_no'])
+                headers = tenant_header['headers']
+                data_size = topological_inventory_data(_, source_id, dest,
+                                                       headers, thread)
+                if data_size > ALERTIBLE_DATE_SIZE:
+                    prometheus_metrics.METRICS['data_size_large'].labels(
+                        account=tenant_header['acct_no']
```

Comment:
This labelling with `acct_no` is against the guidance of prometheus https://prometheus.io/docs/practices/naming/